Formula/tailwindcss-language-server.rb

```diff
@@ -0,0 +1,49 @@
+require "language/node"
+
+class TailwindcssLanguageServer < Formula
+  desc "LSP for TailwindCSS"
+  homepage "https://github.com/tailwindlabs/tailwindcss-intellisense/tree/HEAD/packages/tailwindcss-language-server#readme"
+  url "https://registry.npmjs.org/@tailwindcss/language-server/-/language-server-0.0.11.tgz"
+  sha256 "a46498dea8a2bd36c9596b160e60412a250858a8ba8d8830c7583a5d1b8e89b9"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Replace universal binaries with their native slices
+    (libexec/"lib/node_modules/@tailwindcss/language-server/bin").glob("*.node").each do |f|
+      next if f.arch == Hardware::CPU.arch
+
+      if OS.mac? && f.universal?
+        deuniversalize_machos f
+      else
+        rm f
+      end
+    end
```

Comment:
Is there no way for upstream to build arch specific binaries?

Reply:
cc @bradlc

Reply:
@SMillerDev doesn't look like that will be possible in the near term.

Is that a blocker to move forward?

Reply:
@SMillerDev friendly ping

Reply:
Currently the main blocker is the failing build, but I'm also not very happy that it needs this to install cleanly on mac.

Reply:
I'll get that failing build fixed and then I'll reach out to the tailwindcss maintainers once more, thank you.

Reply:
@SMillerDev To clarify, are you stating you'd like the universal binary pre split?

The code here also deletes the files from non matching architectures, so I'm not sure which or both is not good.

Reply:
Homebrew has no use for universal binaries, since your whole install is architecture dependent. Shipping universal binaries is a waste of space and harder to maintain, so it shouldn't be done and that should start at the source.

Reply:
Additionally, shipping pre-build binaries conflicts with https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae